### PR TITLE
Updates UI for room creation and room view

### PR DESCRIPTION
**Pull request by DustinArmstrong**
Screen Shot 2020-07-01 at 15 44 57-fullpage
Screen Shot 2020-07-01 at 15 53 02-fullpage
Screen Shot 2020-07-01 at 16 33 07-fullpage

### CODE REVIEW

File: lib/littlechat_web/live/room/show_live.ex [hunk 1]
```diff
@@ -28,43 +29,72 @@ defmodule LittlechatWeb.Room.ShowLive do
...
+        <%= for request <- @offer_requests do %>
         <span phx-hook="HandleOfferRequest" data-from-user-uuid="<%= request.from_user.uuid %>"></span>
-      <% end %>
-    </div>
+        <% end %>
+      </div>
+    </main>
+    <footer class="footer">
+      <p>A fun side-project of the ruby &amp; elixir consulting firm Littlelines, LLC in Ohio &copy;2020</p>
+    </footer>
```
Comment (JesseHerrick) [inline lib/littlechat_web/live/room/show_live.ex:91]: Can we move this to the live layout?

File: lib/littlechat_web/live/room/new_live.ex [hunk 2]
```diff
@@ -7,19 +7,28 @@ defmodule LittlechatWeb.Room.NewLive do
...
+          <%= error_tag f, :slug %>
+        </label>
 
-        <%= submit "Save" %>
+        <%= submit "Start Call", class: "button" %>
       <% end %>
-    </div>
+    </main>
+    <footer class="footer">
+      <p>A fun side-project of the ruby &amp; elixir consulting firm Littlelines, LLC in Ohio &copy;2020</p>
```
[the remark this answers is not in the document]
Comment (JesseHerrick) [reply lib/littlechat_web/live/room/new_live.ex:30]: Can we also move this to the footer?

**Comment by JesseHerrick** [comment]
Looks great overall, @DustinArmstrong. I just have a few changes. Also, the changes to the root route made one of the tests fail. I can help with that if you need it.

File: assets/package.json [hunk 3]
```diff
@@ -17,13 +17,17 @@
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
+    "@fortawesome/fontawesome-free": "^5.13.1",
     "babel-loader": "^8.0.0",
     "copy-webpack-plugin": "^5.1.1",
     "css-loader": "^3.4.2",
-    "sass-loader": "^8.0.2",
-    "node-sass": "^4.13.1",
+    "fibers": "^3.1.0",
```
[the remark this answers is not in the document]
Comment (DustinArmstrong) [reply assets/package.json:24]: `npm WARN sass-loader@8.0.2 requires a peer of fibers@>= 3.1.0 but none is installed. You must install peer dependencies yourself.`

[the remark this answers is not in the document]
Comment (DustinArmstrong) [reply assets/package.json]: Here are more details on what it does: https://www.npmjs.com/package/sass#api

**Pull request merged**